tst/tst.yml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
```

Comment:
not sure if we should merge it, and if so, let's not use abbreviations. 
Maybe something like "examples/pod.yml" because it's not a test really :) It's just a fixture for a manual test or an example.